src/plugin/featureaction/ui/editfeatureaction.js

```diff
@@ -32,27 +32,24 @@ goog.require('os.ui.im.action.EventType');
 goog.require('os.ui.im.action.editFilterActionDirective');
```

Comment:
Can you undo the indentation changes in this file? Some lines were split into multiple, when the original line doesn't exceed 120 chars.